Gems/Atom/RHI/Code/Source/RHI/CpuProfilerImpl.cpp

```diff
@@ -218,6 +232,18 @@ namespace AZ
             return m_enabled;
         }
 
+        const CachedTimeRegion::GroupRegionName& CpuProfilerImpl::InsertDynamicName(const char* groupName, const AZStd::string& regionName)
+        {
+            AZStd::scoped_lock lock(m_dynamicNameMutex);
+            AZ_Warning("CpuProfiler", m_dynamicRegionNameSet.size() < 16384, "Stored dynamic region names are accumulating. Consider removing a AZ_ATOM_PROFILE_DYNAMIC invocation.");
+            auto [regionNameItr, wasRegionInserted] =  m_dynamicRegionNameSet.insert(regionName);
```

Comment:
since wasRegionInserted and wasGroupRegionInserted are not used I wonder if you get a "not used" warning from the compiler

Reply:
I think that unused variable warnings don't apply in the case of structured bindings - we'll see if AR throws any issues.